src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.cs

```diff
@@ -460,6 +471,7 @@ public ProcessPriorityClass PriorityClass
         {
             get
             {
+                CheckDisposed();
```

Comment:
This shouldn't throw if `_priorityClass` is populated, that's a potential breaking change.